noether_gui/include/noether_gui/widgets/configurable_tpp_pipeline_widget.h

```diff
@@ -22,7 +22,11 @@ class ConfigurableTPPPipelineWidget : public QWidget
 {
   Q_OBJECT
 public:
-  ConfigurableTPPPipelineWidget(boost_plugin_loader::PluginLoader loader, QWidget* parent = nullptr);
+  ConfigurableTPPPipelineWidget(boost_plugin_loader::PluginLoader loader,
+                                QWidget* parent = nullptr,
+                                QAction* load_action = nullptr,
+                                QAction* save_action = nullptr,
+                                QAction* save_as_action = nullptr);
```

Comment:
Per the comment in the source file, let's not add the action pointers to the constructor